### PR TITLE
Fix new table url

### PR DESCRIPTION
**Pull request by TheBlackBush**
They changed the way to retrive the url for the date table update url, as now its with js function

### CODE REVIEW

File: timewatcher/timewatcher.py [hunk 1]
```diff
@@ -14,6 +14,9 @@
 from . import consts
 from . import data
 
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```
Comment (birkagal) [inline timewatcher/timewatcher.py]: Why do we need this? this package doesn't really use a `logger` so we don't see the urlib3 logs anyway.. (?)

File: timewatcher/timewatcher.py [hunk 2]
```diff
@@ -59,7 +62,7 @@ def _login(
                 "comp": self.config.company_number,
                 "name": self.config.employee_number,
                 "pw": self.config.employee_password,
-            },
+            }, verify=False,
```
Comment (birkagal) [inline timewatcher/timewatcher.py]: Why do we need this? From what iv'e seen the change fix the issue even without disabling this..

**Review by birkagal — CHANGES_REQUESTED** [review]
Hey Idan! thanks for the contribution. I actually saw this right after I implemented my "quick and dirty" fix but I'm loving the effort.
I left some comments on the `urlib3` and `requests` stuff but overall appriciate the effort and would love to merge this!

File: timewatcher/timewatcher.py (hunk 1 above)
Comment (TheBlackBush) [reply timewatcher/timewatcher.py]: We can remove it, it just for me logging

File: timewatcher/timewatcher.py (hunk 2 above)
Comment (TheBlackBush) [reply timewatcher/timewatcher.py]: Sometimes you are behind firewall and you cant verify the host, so that argument will ignore it

**Pull request closed without merging**